server/controllers/getProfile/index.js

```diff
@@ -0,0 +1,20 @@
+const { getProfile } = require('../../database/queries');
+
+const getProfileData = async (req, res, next) => {
+  const {
+    user: { id },
+  } = req;
```

Comment:
Is the userId guaranteed to always be set on `req`? It might be a good idea to add an if-statement to check that it's truthy and throw an error if not.